components/eamxx/src/share/io/tests/io_alias.cpp

```diff
@@ -0,0 +1,105 @@
+#include <catch2/catch.hpp>
```

Comment:
I would also add a test where we actually generate an nc file via our output classes, to verify that AtmosphereOutput uses the aliasing correctly.

Reply:
But, to be clear, I do value the unit test of the aliasing functions.